From f28a96fadc0997687deaa01652ad80b37b08d657 Mon Sep 17 00:00:00 2001
From: Daniel Erez <derez@redhat.com>
Date: Tue, 24 Jul 2018 19:16:10 +0300
Subject: [PATCH] v2v: rhv plugin - fix DC search string

Search for DC by 'storage.name=' to make it explicit.
I.e. "storage=" uses regex, so similar names can be
found in the search query. For example, searching for
a domain named FCSD, will find FCSD1 as well.
---
 v2v/rhv-upload-plugin.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/v2v/rhv-upload-plugin.py b/v2v/rhv-upload-plugin.py
index 931fcfaa2..8274f1ee1 100644
--- a/v2v/rhv-upload-plugin.py
+++ b/v2v/rhv-upload-plugin.py
@@ -72,7 +72,7 @@ def find_host(connection):
     system_service = connection.system_service()
     storage_name = params['output_storage']
     data_centers = system_service.data_centers_service().list(
-        search='storage=%s' % storage_name,
+        search='storage.name=%s' % storage_name,
         case_sensitive=False,
     )
     if len(data_centers) == 0:
-- 
2.20.1

